src/libUtils/DnsUtils.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2021 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#ifndef ZILLIQA_SRC_LIBUTILS_DNSUTILS_H_
+#define ZILLIQA_SRC_LIBUTILS_DNSUTILS_H_
+
+#include <string>
+#include <vector>
+#include "common/BaseType.h"
+
+enum class DnsListType : int {
+  UPPER_SEED = 0,
+  L2L_DATA_PROVIDERS,
+  MULTIPLIERS
+};
+
+void InitDnsCacheList();
```

Comment:
Can you add some level of scoping here? Like namespace or class? So we can call the methods like `DnsUtils::GetPubKeyUrl` or (if needed) `DnsUtils::GetInstance().GetPubKeyUrl`